packages/react/src/components/form/form.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Form } from ".";
+import { ConfigurationProvider } from "../../context/config-context";
+
+const meta: Meta<typeof Form> = {
+  component: Form,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Form>;
+
+export const Primary: Story = {
+  render: () => (
+    <ConfigurationProvider>
+      <Form
+        middleware={({ user }) => {
+          console.log("middleware - user: ", user);
+          return new Promise((resolve) => {
+            resolve(user);
+          });
+        }}
```

Comment:
I think we should probably remove this, it's quite an advanced setting and doesn't feel like it's contributing much here.